frl_vehicle_msgs/msg/UwGliderStatus.msg

```diff
@@ -35,5 +35,5 @@ float32 rudder_angle
 # Estiamted battery pack position in m.
 float32 battery_position
 
-# Estiamted ballast tank pump position in m.
-float32 ballast_position
+# Estimated bouyancy engine pumped volume in L. Positive goes up.
```

Comment:
Recommend staying consistent with the glider hardware interface, so use cm^3.

Is a value of zero neutral?

Reply:
Yes, I believe for a properly ballasted glider that 0 is neutrally buoyant.